modules/standard/List.chpl

```diff
@@ -186,6 +186,13 @@ record list {
     length = 0;
   }
 
+  /*
+    Destructor
+   */
+  proc ~list(){
```

Comment:
Also, there's a TODO above on destroy() which looks like it could be removed now.